src/components/Attachments/AttachmentsReducer.js

```diff
@@ -0,0 +1,107 @@
+import { handleActions } from 'redux-actions';
+import dotProp from 'dot-prop-immutable';
+import * as Actions from './AttachmentsActions';
+
+/**
+ * Use it for debug when styling
+ */
+const debug = false;
+
+const debugDefaultState = {
+  uploads: [
+    {
+      file: {
+        name: 'File uploaded',
+      },
+      status: 'done',
+      result: {
+        id: 'bla bla',
+      },
+    },
+    {
+      file: {
+        name: 'File uploading',
+      },
+      status: 'uploading',
+      progress: 0.47,
+    },
+    {
+      file: {
+        name: 'File error',
+      },
+      status: 'error',
+      error: {
+        message: 'Something went wrong',
+      },
+    },
+    {
+      file: {
+        name: 'File initializing',
+      },
+    },
+  ],
+};
+
+const normalDefaultState = {
+  uploads: [],
+};
+
+const defaultState = debug ? debugDefaultState : normalDefaultState;
+
+function updateFile(updates) {
+  return (state, { payload: { file, uploadFolderId, ...payload } }) =>
```

Comment:
`{ payload: { file, uploadFolderId, ...payload } ` - payload is used 2 times. I think the 2nd time it's used it's not the full payload, but only a part of it? So should be named differently?

Reply:
will replace it to `restPayload`